pages.it/common/az.md

```diff
@@ -2,28 +2,36 @@
 
 > Strumento ufficiale da linea di comando per Microsoft Azure.
 > Alcuni comandi aggiuntivi, come `login`, hanno la propria documentazione.
-> Maggiori informazioni: <https://learn.microsoft.com/cli/azure>.
+> Maggiori informazioni: <https://learn.microsoft.com/cli/azure/reference-index>.
 
 - Effettua il log in ad Azure:
 
 `az login`
 
-- Gestisci la tua iscrizione ad Azure:
+- Gestisce il tuo abbonamento ad Azure:
 
 `az account`
 
 - Elenca tutti i dischi Azure Managed Disks:
 
 `az disk list`
 
-- Elenca tutte le virtual machine Azure:
+- Elenca tutte le macchine virtuali di Azure:
 
 `az vm list`
 
-- Gestisci i servizi Azure Kubernetes:
+- Gestisce i servizi Azure Kubernetes:
 
 `az aks`
 
-- Gestisci le risorse della rete di Azure:
+- Gestisce le risorse della rete di Azure:
 
 `az network`
+
+- Avvia la modalità interattiva:
+
+`az interactive`
+
+- Mostra la descrizioen dei comandi:
```

Comment:
```suggestion
- Mostra la descrizioni dei comandi:
```

Reply:
@gutjuri thanks a lot for your message I have just update the last commit